finder-app/writer.d

```diff
@@ -0,0 +1,71 @@
+writer.o: writer.c \
```

Comment:
@rajatchaple please don't check these files in, add to .gitignore

Reply:
added to .gitignore